tools/publish-packages.js

```diff
@@ -20,21 +21,23 @@ const defaultOptions = {
   exclude: null,
 };
 
+const ROOT_DIR = path.dirname(__dirname);
+
+// sed command differs between platforms
+const SED = os.platform() === 'linux' ? 'sed' : 'gsed';
+
 // configs for other packages that are not unimodules
 const otherPackages = [
   {
     libName: 'expo',
-    dir: `${process.env.EXPO_UNIVERSE_DIR}/exponent/packages/expo`,
+    dir: `${ROOT_DIR}/packages/expo`,
   },
 ];
 
-// list of default package owners - script will ensure all of them are owners of the published packages
-const owners = [
-  'tsapeta',
-  'sjchmiela',
-  'terribleben',
-  'esamelson',
-  'jesseruder'
+// list of teams owning the packages - script will ensure all of these teams are owning published packages
+const teamsWithReadWriteAccess = [
```

Comment:
Don't we use `CONSTANTS_NOTATION` for constants?
```suggestion
const TEAMS_WITH_RW_ACCESS = [
```